config/database.php

```diff
@@ -51,8 +51,8 @@
             'password'       => DB_PASSWORD,
             'unix_socket'    => env('DB_SOCKET', ''),
             'charset'        => DB_CHARSET,
-            'collation'      => DB_COLLATE,
-            'prefix'         => $table_prefix ?? $GLOBALS['wpdb']->prefix,
+            'collation'      => $GLOBALS['wpdb']->collate,
+            'prefix'         => $GLOBALS['wpdb']->prefix,
```

Comment:
@QWp6t What are your thoughts on setting the `prefix` to just `''` instead of using the typical `wp_` prefix? It seems like it would make sense for things coming from Acorn to not use that prefix

For example, the default Laravel users migration example conflicts with the `wp_users` table

Reply:
I think having the WordPress prefix makes sense here, actually.

For non-WordPress database stuff, we should be encouraging the use of the [`mysql`](https://github.com/roots/acorn/blob/main/config/database.php#L64-L82) connection instead of the [`wordpress`](https://github.com/roots/acorn/blob/main/config/database.php#L46-L62) connection. The idea with the `wordpress` connection would be to provide some type of ORM for WordPress, such as through the use of Corcel.

I'm open to any pushback on that idea, though, if you think I'm being a dumdum. :+1:

Reply:
@QWp6t That makes sense :+1: